src/components/Accordion.js

```diff
@@ -88,7 +88,8 @@ export const Accordion = props => {
       width="100%"
       {...rest}
     >
-      <Card
+      <Touchable
```

Comment:
@injas-bigbinary We need to use the Touchable component from the react-native. The one we have in the component doesn't support nested clicks.

Reply:
@Tsudhishnair Updated